backend/src/test/java/com/imsnacks/Nyeoreumnagi/weather/WeatherBriefingTest.java

```diff
@@ -52,11 +55,15 @@ class WeatherBriefingTest {
     @Mock
     private FarmRepository farmRepository;
     @Mock
+    private MyCropRepository myCropRepo;
```

Comment:
P2: 이렇게 변수명 통일하면 좋을 것 같아요 ㅎㅎ

```suggestion
    private MyCropRepository myCropRepository;
```